suite-native/accounts/src/components/AccountsListGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { Box } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { AccountsRootState, selectAccountsByNetworkSymbols } from '@suite-common/wallet-core';
+import { NetworkSymbol } from '@suite-common/wallet-config/libDev/src';
```

Comment:
whoopsie `/libDev/src`

Reply:
@juriczech it is still there

Reply:
https://github.com/trezor/trezor-suite/pull/7149/commits/eff4135b3fe9d660157249245a39d34c6c0ebeff